src/hotspot/cpu/aarch64/assembler_aarch64.hpp

```diff
@@ -1930,31 +1930,17 @@ void mvnw(Register Rd, Register Rm,
     data_processing(op31, type, opcode, Vd, Vn);        \
   }
 
-private:
-  INSN(i_fmovs, 0b000, 0b00, 0b000000);
-public:
-  INSN(fabss, 0b000, 0b00, 0b000001);
-  INSN(fnegs, 0b000, 0b00, 0b000010);
+  INSN(fmovs,  0b000, 0b00, 0b000000);
+  INSN(fabss,  0b000, 0b00, 0b000001);
+  INSN(fnegs,  0b000, 0b00, 0b000010);
```

Comment:
There's unnecessary whitespace here.
```suggestion
  INSN(fmovs, 0b000, 0b00, 0b000000);
  INSN(fabss, 0b000, 0b00, 0b000001);
  INSN(fnegs, 0b000, 0b00, 0b000010);
```

Reply:
Thanks for your review.
It's a style issue. I added one whitespace for `fmov/fabs/fneg` and the later `fcvt` so as to align the arguments to those of `fsqrt`. I saw such a style in several other sites in this header.

If you don't like it, I can remove the added whitespace. But I guess you may also want me to remove the whitespace I added for the `fcvt` below.

Reply:
I see. You're right, we're not consistent about this in the file.

 There are advantages and disadvantages. On the one hand it's a little easier to read, but on the other it's more work to maintain and tends to get eroded over time.

Whitespace-only changes as part of a larger patch can be hard to review , because they ca hide the substantive changes.

Please take out the added whitespace in this patch, and submit just the substantive changes. I will think about what to do to make this file consistent.

Reply:
Updated in the latest revision. Thanks.